system-test/storage.ts

```diff
@@ -880,18 +880,19 @@ describe('storage', () => {
 
     let bucket: Bucket;
 
-    before(() => {
-      bucket = storage.bucket('storage-library-test-bucket');
+    before(async () => {
```

Comment:
@JustinBeckwith / @AVaksman best practice question: should this `before` clause be returning something? or is it okay the way it is?

Reply:
IMHO it is correct as it is right now.

Reply:
Nah this is fine